CHANGELOG.md

```diff
@@ -20,11 +30,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ### Added
 - Use the new repmgr preload library name [(#300)](https://github.com/ManageIQ/manageiq-pods/pull/300)
```

Comment:
This is in Hammer

Reply:
As are #303, #270, #263, #261, and #256

Reply:
Some of these were a bit hard to catch because they were backported a bit unconventionally since we branched Hammer from Gaprindashvili in this repo.